### PR TITLE
Fix Type 'FlutterWindow' not found. (#450)

### PR DESCRIPTION
**Pull request by AndromedaX7**
Flutter version 3.7.0-23.0.pre.22 on channel master

### CODE REVIEW

File: example/ios/Flutter/flutter_export_environment.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/Users/lizhuoyuan/fvm/versions/stable"
-export "FLUTTER_APPLICATION_PATH=/Users/lizhuoyuan/Development/Project/flutter_screenutil/example"
+export "FLUTTER_ROOT=/home/juzi/flutter"
+export "FLUTTER_APPLICATION_PATH=/home/juzi/workspaces/flutter_screenutil/example"
```
Comment (rupinderjeet) [inline example/ios/Flutter/flutter_export_environment.sh:4]: Hi, I think this change should be reverted back.

Comment (AndromedaX7) [reply example/ios/Flutter/flutter_export_environment.sh]: reverted.

**Comment by lizhuoyuan** [comment]
Personal computer configuration

**Comment by zhushenwudi** [comment]
我也在flutter master分支上出现了上述问题
[✓] Flutter (Channel master, 3.7.0-29.0.pre.21, on macOS 13.0 22A380 darwin-arm64, locale zh-Hans-CN)
将FlutterWindow改为FlutterView即可正确编译

**Comment by lizhuoyuan** [comment]
应该会在下午发布一个beta版本
给使用非stable版本的用户使用

**Pull request closed without merging**